DataScience/AisAudioLabeler.py

```diff
@@ -564,13 +580,16 @@ def plot_histogram(ais, max_n_ships):
     None
 
     """
-    fig, axs = plt.subplots()
+    _, axs = plt.subplots(figsize=(10,9), dpi=100)
+    underway_dists = ais.loc[(ais["shipcount_uw"] <= max_n_ships) & (ais["shipcount_uw"] > 0)]["distance"]
```

Comment:
Right, need to exclude values of distance at which no ships are reporting underway.